src/libraries/System.Private.CoreLib/src/System/Runtime/ExceptionServices/ExceptionHandling.cs

```diff
@@ -28,5 +38,22 @@ public static void SetUnhandledExceptionHandler(Func<Exception, bool> handler)
                 throw new InvalidOperationException(SR.InvalidOperation_CannotRegisterSecondHandler);
             }
         }
+
+        /// <summary>
+        /// Raises the <see cref="AppDomain.UnhandledException"/> event.
+        /// </summary>
+        /// <param name="exception">Exception to pass to event handlers.</param>
+        /// <remarks>
+        /// This method will raise the <see cref="AppDomain.UnhandledException"/>
+        /// event and then return.
+        ///
+        /// It will not raise the the handler registered with <see cref="SetUnhandledExceptionHandler"/>.
+        /// </remarks>
+        public static void RaiseAppDomainUnhandledExceptionEvent(object exception)
```

Comment:
I missed this in the API review. This is `object` rather than `Exception` because technically the runtime allows anything to be thrown?

Reply:
Yes, both. The former was compelling, but the latter pushed us over the edge to say it should be `object`.